src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,25 @@
+<header>
+  <div class="container" *ngIf="signedIn; else elseBlock">
+    <a class="ponder-name">Ponder</a>
+    
+    <div class="search-bar">
+      put search bar component here
+    </div>
+      
+    <nav>
+      <ul>
+        <li><button mat-icon-button aria-label="Create Set Button with a plus icon" class="circle-button create">
+          <mat-icon class="symbol add">add</mat-icon>
+        </button></li>
+        <li><button mat-icon-button aria-label="Logout Button with a exit door icon" class="circle-button logout">
+          <mat-icon class="symbol logout">logout</mat-icon>
+        </button></li>
+      </ul>
```

Comment:
Are the class labels "create", "logout", and "add" necessary? There are no additional styles defined for these classes. If they are needed for something (e.g. planned unit tests or page navigation), keep them. Otherwise, I think you can safely delete them.

Reply:
They are not. I will remove them.